staff/doboapps/css-layout-simple/adaptive/styles/mobile.css

```diff
@@ -0,0 +1,25 @@
+html,body{
```

Comment:
TIP some css rules repeat in mobile and desktop. may you group them in main.css.

Reply:
Thank you for the observation. It is already changed. 😅